mockgen/internal/tests/const_array_length/input.go

```diff
@@ -0,0 +1,8 @@
+//go:generate mockgen -package const_length -destination mock.go -source input.go
+package const_length
+
+const C = 2
+
+type I interface {
+	F() [C]int
```

Comment:
Would you add another function to test the other case for arrays: `Bar() [2]int`